### ISSUE TITLE
Not work in Django 1.10

### ISSUE THREAD

**Issue by shamanis** [Alir3z4/django-cuser#17]
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7f9ba3fea378>

Traceback (most recent call last):

  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(_args, *_kwargs)
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/core/management/commands/runserver.py", line 142, in inner_run
    handler = self.get_handler(_args, *_options)
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/contrib/staticfiles/management/commands/runserver.py", line 27, in get_handler
    handler = super(Command, self).get_handler(_args, *_options)
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/core/management/commands/runserver.py", line 64, in get_handler
    return get_internal_wsgi_application()
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/core/servers/basehttp.py", line 49, in get_internal_wsgi_application
    return import_string(app_path)
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/utils/module_loading.py", line 20, in import_string
    module = import_module(module_path)
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/importlib/**init**.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/shaman/Work/Projects/ufanet_ru/configs/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/core/wsgi.py", line 14, in get_wsgi_application
    return WSGIHandler()
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/core/handlers/wsgi.py", line 153, in __init__
    self.load_middleware()
  File "/home/shaman/Work/envs/ufanet_ru/lib/python3.4/site-packages/django/core/handlers/base.py", line 82, in load_middleware
    mw_instance = middleware(handler)
TypeError: object() takes no parameters

For fix:

```
from django.utils.deprecation import MiddlewareMixin
from cuser.middleware import CuserMiddleware


class CuserCustomMiddleware(MiddlewareMixin, CuserMiddleware):
    pass
```

It's work!

**Comment by alutsu** [comment]
where do I put this piece of code?
Thanks

**Comment by Alir3z4** [comment]
@alutsu you need to  have it in here https://github.com/Alir3z4/django-cuser/blob/master/cuser/middleware.py#L6

Replace that line with `class CuserCustomMiddleware(MiddlewareMixin, CuserMiddleware):`

**Comment by alutsu** [comment]
@Alir3z4 
Thank you for your reply, but it works only with `class CuserCustomMiddleware(MiddlewareMixin):` because de CuserMiddleware didn't exists anymore and I changed the settings.py from `cuser.middleware.CuserMiddleware` to `cuser.middleware.CuserCustomMiddleware`

Thank you!!

**Comment by luzfcb** [comment]
@alutsu please, try https://github.com/Alir3z4/django-cuser/pull/19

**Comment by Alir3z4** [comment]
Fixed in #19
